docs/svelte/overview.md

```diff
@@ -1,8 +1,69 @@
 ---
 id: overview
-title: Svelte Query (Coming Soon)
+title: Svelte Query
```

Comment:
@DamianOsipiuk should this `ref` the react docs and replace some parts like the vue docs do?

https://github.com/TanStack/query/blob/fa04a1d26b2900163ffb2e2ae238735fed4dddaa/docs/vue/overview.md#L1-L6

Reply:
Ideally it should if content is appropriate.

Reply:
Can we do this in a follow up @lachlancollins ?